tests/integration_cases/test_raw_node.rs

```diff
@@ -57,22 +57,22 @@ fn conf_change(t: ConfChangeType, node_id: u64) -> ConfChange {
     cc
 }
 
-fn cmp_ready(
```

Comment:
I don't understand why this change. This is less flexible (you can't assert `!cmp_ready(...)` etc).

Is it required? I would hate to see a `must_cmp_ready_false(...)` or something be added in the future :(

Reply:
The reason about do this change is there are many assertions in the function, and they are chained with `&&`. So if one condition fails, we can't know which one it is. But if we do `assert` one by one in the function, we can know which condition fails easily.

Reply:
Ok, let's hope if we need such a functionality in the future we will be clever and add a boolean flag instead of re-implement :)